website/src/routes/publish.lazy.tsx

```diff
@@ -205,7 +205,7 @@ function DeploySubgraph({
     defaultValues: {
       description: subgraphManifest?.parsed.description,
       subgraphImage:
-        'https://api.thegraph.com/ipfs/api/v0/cat?arg=QmdSeSQ3APFjLktQY3aNVu3M5QXPfE9ZRK5LqgghRgB7L9',
+        'https://ipfs.thegraph.com/ipfs/api/v0/cat?arg=QmdSeSQ3APFjLktQY3aNVu3M5QXPfE9ZRK5LqgghRgB7L9',
```

Comment:
a) There's no https://ipfs.thegraph.com/ipfs/api/v0
b) This should use the IPFS Gateway URL anyways: https://ipfs.thegraph.com/ipfs/QmdSeSQ3APFjLktQY3aNVu3M5QXPfE9ZRK5LqgghRgB7L9